### PR TITLE
[nrf noup] dts: Use SoftDevice Controller as default controller

### PR DESCRIPTION
**Pull request by rugeGerritsen**
In the nRF Connect SDK the SoftDevice Controller is the default Bluetooth controller.
This PR  uses the SoftDevice Controller specific device tree binding for this purpose.
    
The existing device tree binding for the open source link layer is kept but disabled. This makes it possible to enable it using a device tree overlay or a snippet.

manifest-pr-skip

### CODE REVIEW

**Review by cvinayak — APPROVED** [review]
QUICK-ACK, but will depend on DTS experts approvals.

**Review by nordicjm — COMMENTED** [review]
OK once file is moved to correct commit

File: dts/arm/nordic/nrf52832.dtsi [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 / {
 	chosen {
-		zephyr,bt-hci = &sdc_controller;
+		zephyr,bt-hci = &bt_hci_sdc;
```
[the remark this answers is not in the document]
Comment (rugeGerritsen) [reply dts/arm/nordic/nrf52832.dtsi:8]: Fixed

**Review by nordicjm — APPROVED** [review]

**Comment by rugeGerritsen** [comment]
Rebased

**Pull request merged**